spec/acceptance/parameter_target/negative/negative_acl_to_symlink_spec.rb

```diff
@@ -37,40 +37,34 @@
         require => File['#{target}']
       }
 
-      exec { 'Create Windows Symlink':
-        command => '#{mklink_command}',
-        creates => '#{target_symlink}',
-        cwd     => '#{target_parent}',
-        require => User['#{user_id}']
+      file { '#{target_symlink}':
+        ensure => link,
+        target => '#{target}',
+        require => User['#{user_id}'],
       }
 
       acl { "#{target_symlink}":
         permissions  => [
           { identity => '#{user_id}', rights => ['full'] },
         ],
-        require      => Exec['Create Windows Symlink']
+        require      => File['#{target_symlink}']
       }
     MANIFEST
   end
 
-  windows_agents.each do |agent|
-    context "on #{agent}" do
-      it 'applies manifest' do
-        execute_manifest_on(agent, acl_manifest, debug: true) do |result|
-          assert_no_match(%r{Error:}, result.stderr, 'Unexpected error was detected!')
-        end
-      end
-
-      it 'verifies ACL rights' do
-        on(agent, verify_acl_command) do |result|
-          assert_no_match(acl_regex, result.stdout, 'Expected ACL was not present!')
-        end
-      end
+  it 'applies manifest' do
+    # not idempotent. complains that "Puppet cannot manage ACLs of symbolic links"
```

Comment:
any ideas on this? It throws the error shown above when attempting to apply the manifest a second time. Is this intentional?

Reply:
This is a known constraint of the module.

> The Windows Provider does not follow Symlinks. Please explicitly manage the permissions of the target

https://github.com/puppetlabs/puppetlabs-acl#Limitations